nussknacker-dist/src/universal/conf/demo-application.conf

```diff
@@ -1,17 +1,19 @@
 base: { include "base-application.conf" }
 
-environment: "demo"
+environment: "local"
```

Comment:
Do we need this file anymore? I think it can be removed?

Reply:
Removed